caom2-repo/src/main/java/ca/nrc/cadc/caom2/repo/client/RepoClient.java

```diff
@@ -611,7 +624,10 @@ private List<DeletedObservation> readDeletedEntityList(DeletionListReader transf
             if (end != null) {
                 surl = surl + "&end=" + df.format(end);
             }
-            log.debug("URL: " + surl);
+
+            if (collectionParams != null) {
```

Comment:
The deleted observation endpoint cannot support extra params because there is no additional info in the DeletedObservation to use to implement it, so here the `collectionParams` should be removed from the inputCollection but just ignored -- not added to the URL.

Reply:
Done, thanks @pdowler !